cms/djangoapps/contentstore/rest_api/v1/tests/test_home.py

```diff
@@ -65,7 +67,7 @@ def test_home_page_response(self):
         }
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertDictEqual(expected_response, response.data)
+        self.assertDictEqual(expected_response, json.loads(json.dumps(response.data)))
```

Comment:
Interesting, I think the `response.data` is in disorder so the json manipulation order it?

Reply:
Nop, the problem was that literally in palm appears  an OrderedDict  so that raises an error because a dict is dict different from an orderedDict

![image](https://github.com/nelc/edx-platform/assets/36200299/4b7c8bc0-d26d-43b2-bc2d-6067200d1c90)